src/App/Pages/Accounts/TwoFactorPage.xaml.cs

```diff
@@ -22,6 +22,8 @@ public partial class TwoFactorPage : BaseContentPage
         private bool _authingWithSso;
         private string _orgIdentifier;
 
+        public Button ContinueButton { get; set; }
```

Comment:
You don't need this anymore after the last round of changes. You can get rid of the corresponding private property as well (`_continueButton`).

Reply:
✅